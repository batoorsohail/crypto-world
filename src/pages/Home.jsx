import Coin from '../components/Coin';

const Home = () => (
  <>
    <Coin />
  </>
);

export default Home;
