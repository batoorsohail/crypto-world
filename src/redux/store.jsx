import { configureStore } from '@reduxjs/toolkit';
import cryptoCoinsReducer from './coin/cryptoSlice';

const store = configureStore({
  reducer: {
    cryptoCoins: cryptoCoinsReducer,
  },
});

export default store;
