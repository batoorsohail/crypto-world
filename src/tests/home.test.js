import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Home from '../pages/Home';

describe('Home component', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <Home />
      </Provider>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
