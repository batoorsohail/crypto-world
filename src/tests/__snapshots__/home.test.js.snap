// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match snapshot 1`] = `
<DocumentFragment>
  <section
    class="input-section"
  >
    <input
      placeholder="Search Crypto"
      type="text"
      value=""
    />
  </section>
  <section
    class="coins-container"
  />
</DocumentFragment>
`;
